include/vcpkg/base/messages.h

```diff
@@ -1088,6 +1090,11 @@ namespace vcpkg
                     "One or more {vendor} credential providers failed to authenticate. See '{url}' for more details "
                     "on how to provide credentials.");
     DECLARE_MESSAGE(FeedbackAppreciated, (), "", "Thank you for your feedback!");
+    DECLARE_MESSAGE(FilesContainAbsolutePath,
```

Comment:
This needs to tell the user about the policy now.

Reply:
I thought the policy should only be used internally to that merging this PR does the break the world. Imho we should not tell the users how to ignore warnings that warns before broken packages.